### DISCUSSION TITLE
Hello from the NERSC software-squad!

### DISCUSSION THREAD

**Discussion by jscook2345** [nwchemgit/nwchem#784] [General]
Hello!

Thanks for all of your work on NWChem! We really appreciate having this software package available for our users at NERSC.

We’re looking to collect comments on how we deploy and support NWChem and we thought it would be a good idea to seek feedback from the development team.

Our current documentation is available here: https://docs.nersc.gov/applications/nwchem/

Does this align with how you’d like to see NWChem used at NERSC?

Additionally, is there anything we could help you with to ensure this software is successfully used at NERSC? Please let us know!

Thanks,

Justin and the NERSC software-squad

**Comment by edoapra** [comment]
@jscook2345 The content I pushed a while ago to https://docs.nersc.gov/applications/nwchem/ is not current for the Perlmutter section.  
Let me try to do a few more tests with the current Shifter image (as discussed in https://github.com/nwchemgit/nwchem/issues/775). Once the tests are done a will submit a pull request to docs.nersc.gov.

**Comment by edoapra** [comment]
Merge request for the NERSC documentation in place  
https://gitlab.com/NERSC/nersc.gitlab.io/-/merge_requests/2202

**Comment by edoapra** [comment]
The NERSC documentation for NWChem was updated with information about the current Shifter information for Perlmutter

https://docs.nersc.gov/applications/nwchem/#slurm-script-for-nwchem-shifter-image-on-perlmutter-cpus

**Comment by jscook2345** [comment]
Thanks so much for your contribution to our docs for our users. I've verified this is merged!

**Reply by edoapra** [reply]
> Thanks so much for your contribution to our docs for our users. I've verified this is merged!

Sure.  
Next month I will push a merge request that removes the Cori bits and adds instructions on how to use NWChem on the GPU partition
